styles/atom-ide-outline.less

```diff
@@ -83,20 +84,65 @@
   .icon {
     contain: @contain_except_size;
     display: inline-block;
-    width: 40px;
+    width: 5ex;
+    font-size: 75%;
     text-align: center;
-    vertical-align: top; // align icon with text vertically
+    vertical-align: middle; // align icon with text vertically
```

Comment:
```suggestion
    vertical-align: middle; // align icon with text vertically
    margin-right: 2ex;
```

Reply:
Not sure it's worth it. Horizontal space is at a bit of a premium. Do we really want to waste 2 characters worth of spacing?

Reply:
This is how it looks on my system:

![image](https://user-images.githubusercontent.com/16418197/111023340-1bd5cd00-839e-11eb-931c-e83b7f9b5a7e.png)

Reply:
Hm. Fonts are monospace, so this shouldn't be an issue. But perhaps I'm missing something. Do you have anything interesting in your local `styles.less` just in case?

Reply:
No, but I am using Dejavu Mono from [fonts](https://atom.io/packages/fonts) package

Reply:
If these suggestions look good on your system, we can move forward with it. We can debug the issue later.

Reply:
Let me figure it out. Really no point in merging this if it breaks with some fonts.

Reply:
Okay, so `4em` will work regardless of the font, but it leaves a little too much spacing for my taste. I'll play around with alignment to see if that helps visually

Reply:
After applying these two suggestions, I tested them with a lot of fonts including the default font families. It works properly now.

Reply:
Okay, I've figured it out. My CSS is somewhat rusty, so I've used `ex` (which is vertical size of lowercase `x`) instead of `ch` (which is horizontal size of `0`). `5ch` on `width` without additional margins should work fine.

Reply:
I've pushed two commits, one reverting your attempt at the fix (sorry). Check if it helps on your system?

Reply:
No, unfortunately. 

![image](https://user-images.githubusercontent.com/16418197/111024329-d74d3000-83a3-11eb-87e4-ab3c3d51c07d.png)

Reply:
You sure you've pulled and restarted? Because this works as expected on my side even with "weird" fonts with very large sizes.
![image](https://user-images.githubusercontent.com/7275622/111024444-de019480-83ef-11eb-8203-4ae75d5110b4.png)
![image](https://user-images.githubusercontent.com/7275622/111024468-fa053600-83ef-11eb-899c-a9244a51f430.png)

Reply:
Yes, I did. 

![image](https://user-images.githubusercontent.com/16418197/111024614-4ecf8f00-83a5-11eb-88a4-daede3a5722c.png)

Reply:
I don't get it :confused: `ch` is by spec exactly the width of one character (for monospace fonts). Ergo the width _should_ be enough for 4 characters with 0.5-character margin on both sides. And it holds pretty much exactly on my side. What's different for you?

My Atom version is:
```
Atom    : 1.55.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
```

Reply:
No idea! 🤷‍♂️ Seems weird. We can ask others to test it too. @UziTech 

```
❯ atom --version

Atom    : 1.55.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
```